src/Infrastructure/MQTT/MqttMessageSender.cs

```diff
@@ -51,5 +52,10 @@ public bool SendMessageToConfig(AppConfigDto configuration)
         {
             throw new NotSupportedException();
         }
+
+        public Task SendMessageToAlarmsAsync(AlarmMessage alarmMessage)
+        {
+            throw new NotImplementedException();
```

Comment:
Why do we not send alarms here? Should this not also be on the telemetry channel? Or would another channel be required?

Reply:
Must have overlooked it as I initially developed the alarms feature whilst using a kafka application host to test it. :)

Reply:
Added